Makefile

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Mainflux
+# SPDX-License-Identifier: Apache-2.0
+
+BUILD_DIR = build
+SERVICES = users things http ws coap lora influxdb-writer influxdb-reader mongodb-writer \
```

Comment:
This Makefile should be changed.

You should use generic lightweight Makefile from `mqtt-bench` tool.